src/processors/InputProcessor.ts

```diff
@@ -1,21 +1,19 @@
 import { AbstractInputProcessor } from "./AbstractInputProcessor";
 import { AsyncAPIInputProcessor } from "./AsyncAPIInputProcessor";
 import { JsonSchemaInputProcessor } from "./JsonSchemaInputProcessor";
-
 import { CommonInputModel } from "../models/CommonInputModel";
 
 /**
  * Main input processor which figures out the type of input it receives and delegates the processing into separate individual processors.
  */
-export class InputProcessor extends AbstractInputProcessor {
-  processors: Map<string, AbstractInputProcessor> = new Map();
+export class InputProcessor {
+  processors: {[key: string]: AbstractInputProcessor} = {};
 
   constructor(){
-    super();
     const asyncAPI = new AsyncAPIInputProcessor();
     this.addProcessor("asyncapi", asyncAPI); 
     const jsonSchema = new JsonSchemaInputProcessor();
-    this.addProcessor("json-schema", jsonSchema);
+    this.addProcessor("default", jsonSchema);
```

Comment:
I could not find any other way to add a default processor while still making it changeable.